prov/efa/src/efa_base_ep.h

```diff
@@ -83,6 +85,51 @@ extern struct fi_provider efa_prov;
 /* maximum name length for shm endpoint */
 #define EFA_SHM_NAME_MAX	   (256)
 
+#define EFA_ABI_VER_MAX_LEN 8
+
+#define EFA_WCE_CNT 1024
+
+#define EFA_EP_TYPE_IS_RDM(_info) \
+	(_info && _info->ep_attr && (_info->ep_attr->type == FI_EP_RDM))
+
+#define EFA_EP_TYPE_IS_DGRAM(_info) \
+	(_info && _info->ep_attr && (_info->ep_attr->type == FI_EP_DGRAM))
+
+#define EFA_DEF_POOL_ALIGNMENT (8)
+#define EFA_MEM_ALIGNMENT (64)
+
+#define EFA_DEF_CQ_SIZE 1024
+#define EFA_MR_IOV_LIMIT 1
+#define EFA_MR_SUPPORTED_PERMISSIONS (FI_SEND | FI_RECV | FI_REMOTE_READ)
+
+/*
+ * Setting ibv_qp_attr.rnr_retry to this number when modifying qp
+ * to cause firmware to retry indefininetly.
+ */
+#define EFA_RNR_INFINITE_RETRY 7
+
+/*
+ * Multiplier to give some room in the device memory registration limits
+ * to allow processes added to a running job to bootstrap.
+ */
+#define EFA_MR_CACHE_LIMIT_MULT (.9)
+
+#define EFA_MIN_AV_SIZE (16384)
+
```

Comment:
MIN_AV_SIZE should not be moved to `efa_base_ep.h` because they are not EP related.